CMakeLists.txt

```diff
@@ -183,8 +184,21 @@ foreach (_variableName ${_variableNames})
 endforeach()
 ]]#
 
-set(DATADIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}")
-set(FULL_DATADIR "${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT_NAME}")
+########### OS X Bundling ###############
+if(APPLE)
+  configure_file(${CMAKE_SOURCE_DIR}/cmake/Info.plist.cmake ${CMAKE_SOURCE_DIR}/Info.plist @ONLY)
```

Comment:
1. instead `if(APPLE)` please use `if(NATIVE_OSX_APP)` in that case it will be still possible to install to /usr/local or other location like in other unix-like systems
1. move cmake/Info.plist.cmake to Info.plist.in, cmake dir for cmake modules only, and it's preferred to add `.in` suffix for templates
1. set destination to ${CMAKE_BINARY_DIR}/Info.plist